internal/lsp/completions/providers/rulerefs.go

```diff
@@ -33,108 +36,159 @@ func (*RuleFromImportedPackageRefs) Run(
 		return nil, nil
 	}
 
+	words := patternWhiteSpace.Split(currentLine, -1)
+	lastWord := words[len(words)-1]
+
 	// some version of a parsed mod is needed here to filter refs to suggest
 	// based on import statements
 	mod, ok := c.GetModule(fileURI)
 	if !ok {
 		return nil, nil
 	}
 
-	refsFromImports := make(map[string]types.Ref)
+	refsForContext := make(map[string]types.Ref)
 
-	for file, refs := range c.GetAllFileRefs() {
-		if file == fileURI {
-			continue
-		}
-
-		for key, ref := range refs {
-			// we are not interested in packages here, only the rules
+	for uri, refs := range c.GetAllFileRefs() {
+		for _, ref := range refs {
+			// we are not interested in packages here, only rules
 			if ref.Kind == types.Package {
 				continue
 			}
 
-			// don't suggest refs of "private" rules, even
-			// if only just by naming convention
+			// don't suggest refs of "private" rules, even if this
+			// is only just a naming convention
```

Comment:
Sneaky move, approving my PR only to correct me later hoping I wouldn't notice 😄

Reply:
Ha, sorry...